incubator/kubeflow/requirements.lock

```diff
@@ -0,0 +1,12 @@
+dependencies:
+- name: ambassador
```

Comment:
We're moving off of ambassador to Istio. This is probably not needed.

Reply:
Good to know. So Istio will be a prereq to kubeflow? :(
It would be nice to allow for ingress options. 
I for one would like to expose the UIs using nginx-ingress for example.

Reply:
Thanks for letting me know.